driver/types.go

```diff
@@ -61,12 +61,25 @@ type (
 	Vector json.RawMessage
 )
 
+// ReadOptions modifies read request behavior.
+type ReadOptions struct {
```

Comment:
At first, I created `ReadOption` as Message type in api.proto but realised, it is not needed there and can be split between `ReadRequestOptions` and `Sort`. So just defining it as struct here. Let me know, if this is not fine.